extensions/likes/src/Listener/SaveLikesToDatabase.php

```diff
@@ -13,13 +13,23 @@
 use Flarum\Likes\Event\PostWasUnliked;
 use Flarum\Post\Event\Deleted;
 use Flarum\Post\Event\Saving;
+use Illuminate\Contracts\Events\Dispatcher;
 
 class SaveLikesToDatabase
 {
+    /**
+     * @param Dispatcher $events
+     */
+    public function subscribe(Dispatcher $events)
+    {
+        $events->listen(Saving::class, [$this, 'whenPostIsSaving']);
+        $events->listen(Deleted::class, [$this, 'whenPostIsDeleted']);
```

Comment:
you forgot to un-static `whenPostIsDeleted`

Reply:
Damn 🤦